ci-operator/step-registry/openshift/e2e/cert-rotation/prepare-for-shutdown/openshift-e2e-cert-rotation-prepare-for-shutdown-commands.sh

```diff
@@ -157,6 +157,42 @@ oc patch configs.imageregistry.operator.openshift.io cluster --type merge --patc
 oc patch OperatorHub cluster --type json \
     -p '[{"op": "add", "path": "/spec/disableAllDefaultSources", "value": true}]'
 
+# Generate custom ingress certificate valid for 10 years
+tenYears=3650
+baseDomain=$(oc get dns.config cluster -o=jsonpath='{.spec.baseDomain}')
+temp_dir=$(mktemp -d)
+openssl req -newkey rsa:4096 -nodes -sha256 -keyout "${temp_dir}/ca.key" -x509 -days ${tenYears} -subj "/CN=$baseDomain" -out "${temp_dir}/ca.crt"
+
+openssl genrsa -out "${temp_dir}/ca.key" 4096
+openssl req -x509 -sha256 -key "${temp_dir}/ca.key" -nodes -new -days ${tenYears} -out "${temp_dir}/ca.crt" -subj "/CN=${baseDomain}" -set_serial 1
+
+oc create configmap custom-ca \
+     --from-file=ca-bundle.crt="${temp_dir}/ca.crt" \
+     -n openshift-config
+
+oc patch proxy/cluster \
+     --type=merge \
+     --patch='{"spec":{"trustedCA":{"name":"custom-ca"}}}'
+
+defaultIngressDomain=$(oc get ingresscontroller default -o=jsonpath='{.status.domain}' -n openshift-ingress-operator)
+cat <<EOZ > "${temp_dir}/tmp.conf"
+[cus]
+subjectAltName = DNS:*.${defaultIngressDomain}
+EOZ
+openssl genrsa -out "${temp_dir}/server.key" 4096
+openssl req -new -key "${temp_dir}/server.key" -subj "/CN=${defaultIngressDomain}" -addext "subjectAltName = DNS:*.${defaultIngressDomain}" -out "${temp_dir}/server.csr"
+openssl x509 -req -days ${tenYears} -CA "${temp_dir}/ca.crt" -CAkey "${temp_dir}/ca.key" -CAserial caproxy.srl -CAcreateserial -extfile "${temp_dir}/tmp.conf" -extensions cus -in "${temp_dir}/server.csr" -out "${temp_dir}/server.crt"
```

Comment:
I don't see where `caproxy.srl` is used, can I omit this parameter `-CAserial`, at least I use it in this way.

Reply:
This would set output file for "-CAcreateserial", otherwise it would use `ca.srl`. We don't really need it, as we create just one certificate (but we'd need that file if we sign more)

Reply:
Sure